### ISSUE TITLE
Action Metrics

### ISSUE THREAD

**Issue by joshua-kim** [ava-labs/hypersdk#1223]
HyperSDK should have some way of exposing aciton-level metrics (count, latency for verify/execute, etc).

**Comment by aaronbuchwald** [comment]
Do you mean by typeID?

**Comment by joshua-kim** [comment]
> Do you mean by typeID?
Yeah, we'll have to parse the type id to route it to the corresponding metric.
